### ISSUE TITLE
Add global hotkey to toggle KeePassXC to/from tray

### ISSUE THREAD

**Issue by ChristopherSchultz** [keepassxreboot/keepassxc#2276]
I believe this was in the original KeePass for Windows, and that the default was something like CTRL-ALT-K. All it did was bring the KeePass application window to the foreground.

This was great when auto-type wasn't appropriate but you wanted to get KeePass up right away (I'm a keyboard guy, so using the mouse to pull it up kinda sucks).

Long ago I moved to Mac + KeePassX and now recently KeePassXC. Since my move to Mac, I've been using BetterTouchTool to register the CTRL-CMD-K hotkey for this purpose, but it would be nice for it to be provided out-of-the-box.

**Comment by sergeevabc** [comment]
Similar crucial [issue][1] was created in **2016**. Still not resolved.

[1]: https://github.com/keepassxreboot/keepassxc/issues/99

**Comment by arigit** [comment]
In linux / gnome, you can setup system shortcut to launch keepassxc with ctrl alt k. You can also set keepassxc to single instance mode. The result is about the same as what you have in in windows with keepass2. Wondering if the same approach would work on mac.
The only annoyance on linux - couldn't find a way yet to replicate the behaviour of ctrl-alt-a that used to work with keepass2 - triggering global auto type from the command line (needed if you don't want to keep keepassxc in the tray all the time)

**Comment by phoerious** [comment]
Duplicate of #99

**Comment by adamency** [comment]
@phoerious This is **not** a duplicate of #99. #99 asks about the toggle feature itself, which is today already possible, but seemingly only via the taskbar icon itself and not the window:
![image](https://user-images.githubusercontent.com/77821695/232324843-38b427c0-feab-4cdf-80a5-2929156be51f.png)

However this current issue is asking for a keyboard shortcut for this feature, which I haven't been able to find. Is there one ? If not, this issue should be reopened.

**Comment by adamency** [comment]
It should be noted that the keyboard shortcut for doing this is `Ctrl+Shift+M`: "*Hide Window*"

(https://keepassxc.org/docs/KeePassXC_UserGuide#_keyboard_shortcuts)

**Comment by DerTee** [comment]
> It should be noted that the keyboard shortcut for doing this is `Ctrl+Shift+M`: "_Hide Window_"
> 
> (https://keepassxc.org/docs/KeePassXC_UserGuide#_keyboard_shortcuts)

No, it's not for the intents of users, because **showing** KeePassXC is the important bit. Making it a toggle to also hide the window, is just a minor convenience. You're referring to the minor part, not the major part of the issue.

**Comment by adamency** [comment]
@DerTee

First, your sentences don't make any sense.
Second you don't even understand what has been talked about in this thread which is exactly what I mentioned, but you're interpreting it your own way because you are that self-involved.
Third, you're not even trying to bring something to the conversation but simply spew out your frustrations like a spoiled child.

You should learn how to use GitHub and how to read threads. What you're talking about is a completely other problem, so open another issue if you want help on this (and it's not even possible to deal with it via the software but only via the OS, but you don't seem knowledgeable enough to understand what I'm referring to here) and stop polluting conversations with out of topic stuff.

**Comment by droidmonkey** [comment]
This is a duplicate request that was closed, thanks
